### ISSUE TITLE
Theme customizations ignored

### ISSUE THREAD

**Issue by wamatt** [buymeasoda/soda-theme#198]
I have followed all instructions in the wiki, and been through all the the troubleshooting. Yet cannot for the life of me, manage to get theme customizations working.

Here is a copy of `Soda Light 3.sublime-theme` in my User dir:

```JSON
[
	{
	    "class": "tab_label",
	    "settings": ["highlight_modified_tabs"],
	    "parents": [{"class": "tab_control", "attributes": ["dirty"]}],
	    "fg": [245, 100, 90] 
	} 
]
````

- Declaration is fine:  `"theme": "Soda Light 3.sublime-theme"` in my `~/Library/Application Support/Sublime Text 3/Packages/User/Preferences.sublime-settings` file.

- The theme itself works fine. It's just customizations are ignored.

No errors in console. However I should point out that it does appear to be reading the user file. As an example, if one places weird characters in the user file eg `[] blah blah` it will throw an error in console

`Error trying to parse file: Unexpected trailing characters in Packages/User/Soda Light 3.sublime-theme:8:3`

**Comment by kevinSuttle** [comment]
I'm also experiencing this.

**Comment by mhogerheijde** [comment]
I found that removing the line `"settings": ["highlight_modified_tabs"]` makes this work!

I.e. using 

```
[
  {
      "class": "tab_label",
      "parents": [{"class": "tab_control", "attributes": ["dirty"]}],
      "fg": [245, 100, 90]
  }
]
```

works like expected for me.

I firstly tried other customisations, by copy pasting from https://github.com/buymeasoda/soda-theme/blob/master/Soda%20Dark%203.sublime-theme and ajusting minor things. Then I found that taking out the line about `setting` made it work.

**Comment by kevinSuttle** [comment]
Nope. I copy/pasted from that file @mhogerheijde linked to, removed all `"settings": ["highlight_modified_tabs"]` lines, and still nothing.

**Comment by mhogerheijde** [comment]
Strange?

```
$ subl --version
Sublime Text Build 3083
```

Theme soda version: 2014.05.07.06.11.27

I'm currently using the _dark theme_:

`Preferences.sublime-settings`

```json
{
    "color_scheme": "Packages/Color Scheme - Default/Monokai.tmTheme",
    "theme": "Soda Dark 3.sublime-theme",
    "..." : "..."
}

```

**Comment by hemendra619** [comment]
I have installed this theme. It got installed, tab color os diffrent now but background color didn't cahnged. 
```
{
	"color_scheme": "Packages/User/SublimeLinter/Monokai (SL).tmTheme",
	"theme": "Soda Light 3.sublime-theme"
}
```
When i delete the color_sheme line, it automatically come back. Seems some conflict with linter color scheme and theme color scheme.

**Comment by yeongjoshua** [comment]
![screen shot 2015-11-05 at 12 53 31 pm](https://cloud.githubusercontent.com/assets/5170481/10960168/935e429a-83bc-11e5-9b31-f835de586d15.png)
I'm having problem with fresh install too

**Comment by rooraggio** [comment]
ditto to all of the above

**Comment by slifin** [comment]
I also experience this, odd because there is one customisation in the file that does take effect, I changed the tabs to be higher, now if I change anything about the file including the tab height no change is made
